src/components/SamlProviderConfiguration/SamlProviderConfigForm.jsx

```diff
@@ -110,46 +185,34 @@ class SamlProviderConfigForm extends React.Component {
       >
         <div className="row">
           <div className="col col-6">
-            <ValidationFormGroup
-              for="enabled"
-            >
-              <label htmlFor="enabled">Enabled</label>
-              <Input
-                type="checkbox"
+            <Form.Group controlId="enabled">
+              <Form.Label htmlFor="enabled">Enabled</Form.Label>
```

Comment:
Same questions about unnecessary `htmlFor` on `Form.Label` and `id` on `Form.Control`